src/OrchardCore.Modules/OrchardCore.Users/Views/Items/UserDeletedEvent.Fields.Design.cshtml

```diff
@@ -0,0 +1,5 @@
+@model OrchardCore.Users.Workflows.ViewModels.UserEventViewModel<OrchardCore.Users.Workflows.Activities.UserDeletedEvent>
+
+<header>
+    <h4><i class="fa fa-user"></i>@Model.Activity.GetTitleOrDefault(() => T["User Deleted"])</h4>
```

Comment:
`fa-user-minus` could be appropriate.
https://fontawesome.com/icons/user-minus?style=solid

Reply:
Very nice, changed.

Reply:
It just occurred to me that I should do the same with fa-user-plus for UserCreatedEvent. Pushed.